lldb/source/Plugins/SymbolFile/CTF/SymbolFileCTF.cpp

```diff
@@ -1032,10 +1032,16 @@ void SymbolFileCTF::FindTypes(const lldb_private::TypeQuery &match,
 
   ConstString name = match.GetTypeBasename();
   for (TypeSP type_sp : GetTypeList().Types()) {
-    if (type_sp && type_sp->GetName() == name) {
-      results.InsertUnique(type_sp);
-      if (results.Done(match))
-        return;
+    if (type_sp) {
```

Comment:
If you're already rewriting this, lets do:
```
if (!type_sp)
  continue;
```
(same applies to the `type_name == name` condition below)